xml/System/String.xml

```diff
@@ -11629,6 +11650,9 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
    
   
 ## Examples  
+
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```

Comment:
There's already a note about this in the remarks of the page

```suggestion
```

Reply:
Done